src/assets/ProjectData.json

```diff
@@ -0,0 +1,47 @@
+[
+    {
+        "featured": true,
+        "display": true ,
+        "name": "Happy Thoughts",
+        "overlayname": "Happy Thoughts",
+        "description": "App to collect happy thoughts, in a twitter-like feed, using React and API.",
+        "tags": ["HTML", "CSS", "JavaScript", "API"],
+        "image": "./assets/happythoughts.png",
+        "netlify": "https://our-happy-thoughts.netlify.app/",
+        "github": "https://github.com/jesshansson/project-happy-thoughts"
+    },
+    {
+        "featured": true,
+        "display": true,
+        "name": "Find-a-Friend survey",
+        "overlayname": "React Survey",
+        "description": "A Typeform-like survey built with React state",
+        "tags": ["HTML", "CSS", "JavaScript", "React"],
+        "image": "./assets/survey.png",
+        "netlify": "https://find-a-friend-survey.netlify.app/",
+        "github": "https://github.com/jesshansson/project-survey"
+    },    
+    {
+        "featured": true,
+        "display": true,
+        "name": "Popular Movies",
+        "overlayname": "Movies",
+        "description": "A website that shows the most popular movies today, built with React components, fetching the data from an API.",
+        "tags": ["HTML", "CSS", "JavaScript", "React"],
```

Comment:
I would also add a API tag to all the projects where we are fetching information from API:s. It's a very valuable thing to have learnt and worth pointing out! (y)